packages/sources/gramchain/CHANGELOG.md

```diff
@@ -0,0 +1,229 @@
+# @chainlink/example-source-adapter
+
+## 1.0.25
+
+### Patch Changes
+
+- Updated dependencies [4d6b8a050]
+  - @chainlink/ea-bootstrap@1.10.4
+  - @chainlink/ea-test-helpers@1.0.1
```

Comment:
Can you actually delete this file, and just run `yarn changeset` to create the changeset for ea-bootstrap? We generate the changelog/versions as a separate step in the development pipeline to generate releases.